src/hyp3_isce2/insar_tops_burst.py

```diff
@@ -201,6 +203,47 @@ def insar_tops_single_burst(
         packaging.upload_product_to_s3(product_dir, output_zip, bucket, bucket_prefix)
 
 
+def insar_tops_multi_burst(
+    reference: Iterable[str],
+    secondary: Iterable[str],
+    swaths: list = [1, 2, 3],
+    looks: str = '20x4',
+    apply_water_mask=False,
+    bucket: Optional[str] = None,
+    bucket_prefix: str = '',
+):
+    ref_ids = [g.split('_')[1] + '_' + g.split('_')[2] + '_' + g.split('_')[4] for g in reference]
+    sec_ids = [g.split('_')[1] + '_' + g.split('_')[2] + '_' + g.split('_')[4] for g in secondary]
+
+    if ref_ids != sec_ids:
+        raise Exception('The reference burst(s) and secondary burst(s) do not match')
```

Comment:
In the interest of using consistent terminology, is "burst ID" an official term? In the [`check_same_burst_ids`](https://github.com/ASFHyP3/hyp3/blob/develop/apps/api/src/hyp3_api/validation.py#L79) hyp3 validator function we define burst ID as `granule['name'].split('_')[1]`, but here we're extracting the ID by combining indices `[1]`, `[2]`, and `[4]`.

Reply:
Also, `Exception` is too general, we probably want `ValueError`.

Reply:
Yeah, "burst id" is a problematic term and I know of at least four different versions of what it could mean. Here, we're getting the minimum information needed to identify a unique burst (burst number, swath and polarization). I've updated the names slightly to try to make this clearer.

Reply:
@forrestfwilliams How about just `ref_number_swath_pol` and `sec_number_swath_pol`?